src/Infrastructure/Persistence/Configuration/MissionConfiguration.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using SSW.CleanArchitecture.Domain.Common;
+using SSW.CleanArchitecture.Domain.Teams;
+
+namespace SSW.CleanArchitecture.Infrastructure.Persistence.Configuration;
+
+public class MissionConfiguration : IEntityTypeConfiguration<Mission>
+{
+    // TODO: Rip out the common pieces that are from BaseEntity (https://github.com/SSWConsulting/SSW.CleanArchitecture/issues/78)
```

Comment:
Is this TODO still relevant?

Reply:
The idea was to see if we can have generic DB config code for `Id` related config.

I'm not sure if it's possible, but worth looking into at some stage.

Reply:
I've added TODO back to the marker :)